packages/@okta/i18n/src/properties/login_ja.properties

```diff
@@ -537,7 +537,7 @@ password.reset.title = Okta\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C
 password.reset.title.generic = \u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8\u3057\u307E\u3059
 password.reset.title.specific = {0}\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8\u3057\u307E\u3059
 password.complexity.requirements = \u30D1\u30B9\u30EF\u30FC\u30C9\u8981\u4EF6: {0}\u3002
-password.complexity.history = \u904E\u53BB\u306B\u4F7F\u3063\u305F\u3053\u3068\u306E\u3042\u308B{0}\u3064\u524D\u307E\u3067\u306E\u30D1\u30B9\u30EF\u30FC\u30C9\u306F\u5229\u7528\u3067\u304D\u307E\u305B\u3093\u3002
+password.complexity.history = \u904E\u53BB\u306B\u4F7F\u3063\u305F\u3053\u3068\u306E\u3042\u308B{0}\u4E16\u4EE3\u524D\u307E\u3067\u306E\u30D1\u30B9\u30EF\u30FC\u30C9\u306F\u5229\u7528\u3067\u304D\u307E\u305B\u3093
```

Comment:
This is the correct fix. Thank you. @TianGan-okta @haishengwu-okta @santhoshbalakrishnan